core/src/main/resources/hudson/model/AllView/noJob.properties

```diff
@@ -24,4 +24,4 @@ newJob=Please <a href="newJob">create new jobs</a> to get started.
 
 login=<a href="{0}/{1}?from={2}">Log in</a> to create new jobs.
 
-signup=If you don\u2019t already have an account, you can <a href="signup">sign-up</a> now.
+signup=If you don\u2019t already have an account, you can <a href="signup">sign-up</a> now.
```

Comment:
@jglick Addressed here https://github.com/jenkinsci/jenkins/pull/2117/commits/7744fbba19ab6533b294b220daf26eb732522c63

I have to verify it with @gusreiber but this file is not related with this PR.